src/app/cube/new-home/help/teach-now/teach-now.component.ts

```diff
@@ -1,15 +1,106 @@
-import { Component, OnInit } from '@angular/core';
+import { AfterViewInit, Component, ElementRef, HostListener, OnInit, ViewChild } from '@angular/core';
+import { Router } from '@angular/router';
+import { LearningObject, Topic } from '@entity';
+import { LearningObjectService } from 'app/cube/learning-object.service';
+import { RelevancyService } from '../../../../core/relevancy.service';
 
 @Component({
   selector: 'clark-teach-now',
   templateUrl: './teach-now.component.html',
   styleUrls: ['./teach-now.component.scss']
 })
-export class TeachNowComponent implements OnInit {
+export class TeachNowComponent implements OnInit, AfterViewInit {
 
-  constructor() { }
+  // Topic variables
+  topics: Topic[] = [];
+  selectedTopic: string;
+
+  // HTML elements
+  @ViewChild('topicScroll') scroll: ElementRef;
+  width: number = window.screen.width;
+
+  // Object variables
+  objects: LearningObject[] = [];
+  loadingObjects = [new LearningObject(), new LearningObject(), new LearningObject(), new LearningObject()];
```

Comment:
Is there a reason for the hardcoded new objects?

Reply:
This is what is used to show loading skeleton learning object cards.